README.md

```diff
@@ -71,65 +72,92 @@ Provides the following HTTP methods for `/load-test` endpoint:
-    ```
-    
-3. Download the dependencies
-    ```bash
-    go mod vendor
-    ```
-
-4. Build Kangal binary
-   ```bash
-   make build
-    ```
-    
-5. Set the environment variables
-    ``` bash
-    export WEB_HTTP_PORT=8080                    # API port for Kangal Proxy
-    export AWS_BUCKET_NAME=YOUR_BUCKET_NAME      # name of the bucket for saving reports
-    export AWS_ENDPOINT_URL=YOUR_BUCKET_ENDPOINT # storage connection parameter
-    export AWS_DEFAULT_REGION=YOUR_AWS_REGION    # storage connection parameter
-    ```
-
-6. Run both Kangal proxy and controller
-    ```bash
-    ./kangal controller --kubeconfig=$KUBECONFIG 
-    ./kangal proxy --kubeconfig=$KUBECONFIG
-    ```
+### Kangal Controller
+Component responsible to manage all the aspects of the performance testing process.
 
-## Contributing
+## Quickstart guide
+This tutorial will guide through Kangal installation process and usage.
+
+### Installing using helm
+First, let's create the Custom Resource Definition by running:
+
+```shell
+$ kubectl apply -f https://raw.githubusercontent.com/hellofresh/kangal/master/charts/kangal/crd.yaml
+```
+
+Now, install the chart using following commands:
+
+```shell
+$ helm repo add kangal https://hellofresh.github.io/kangal
+$ helm install --set environment=dev kangal kangal/kangal
+```
+
+That's it, Kangal should be installed, check if is all correct by running:
+
+```shell
+$ kubectl get pods
+NAME                                 READY   STATUS    RESTARTS   AGE
+kangal-controller-588677b854-r9qcs   1/1     Running   0          44s
+kangal-openapi-ui-7c5dd8997c-jj4mk   1/1     Running   0          44s
+kangal-openapi-ui-7c5dd8997c-vgm8c   1/1     Running   0          44s
+kangal-proxy-7d95c9d65-6t44b         1/1     Running   0          44s
+kangal-proxy-7d95c9d65-75dv4         1/1     Running   0          44s
+```
+
+For more information about the Helm Chart check [charts/kangal/README.md](charts/kangal/README.md).
+
+### Creating first LoadTest
+To run an LoadTest you first need to find Kangal proxy endpoint, use this command:
+
+```shell
+$ kubectl get ingress
+NAME                HOSTS                  ADDRESS     PORTS   AGE
+kangal-openapi-ui   kangal-openapi.local   localhost   80      5m48s
+kangal-proxy        kangal-proxy.local     localhost   80      5m48s
+```
 
-To start contributing, please check [CONTRIBUTING](CONTRIBUTING.md).
+> This is assuming you have a proper configured [Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/), if is not the case you can use [Port Forwarding](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/).
+
+With this information, you are now able to do a request to create the first load test.
+Let's start by downloading an example JMeter test and POST it to Kangal proxy.
+
+```shell
+$ curl -s -O https://raw.githubusercontent.com/hellofresh/kangal/master/examples/constant_load.jmx
```

Comment:
This test will fail for people if they don't have InfluxDB in cluster because this example file was created for our infrastructure and contains influxdbMetricsSender in it. Probably we need to create another example, even simpler than this one.

Reply:
I removed the InfluxDB part from the test :)